src/plugins/initiatives/initiativeEntry.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import {type TimestampMs} from "../../util/timestamp";
+import {type NodeAddressT} from "../../core/graph";
+import {type InitiativeWeight} from "./initiative";
+import {type NodeEntryJson} from "./nodeEntry";
+
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeEntry = {|
+  +id: string, // GUID
+  +title: string,
+  +timestampMs: TimestampMs,
+  +weight: InitiativeWeight,
+  +completed: boolean,
+  // should be user nodes
+  +champions: $ReadOnlyArray<NodeAddressT>,
+  // other initiative nodes
+  +dependencies: $ReadOnlyArray<NodeAddressT>,
```

Comment:
(re comment): I think dependencies can be any nodes, not just initiatives nodes.